components/net/http_cache.rs

```diff
@@ -280,24 +271,19 @@ fn get_expiry_adjustment_from_request_headers(request: &Request, expires: Durati
     };
 
     if let Some(max_age) = directive.max_stale() {
-        return expires + Duration::from_std(max_age).unwrap();
+        return expires + max_age;
     }
     if let Some(max_age) = directive.max_age() {
-        let max_age = Duration::from_std(max_age).unwrap();
         if expires > max_age {
-            return Duration::min_value();
+            return Duration::ZERO;
         }
         return expires - max_age;
     }
     if let Some(min_fresh) = directive.min_fresh() {
-        let min_fresh = Duration::from_std(min_fresh).unwrap();
-        if expires < min_fresh {
-            return Duration::min_value();
-        }
-        return expires - min_fresh;
+        expires.checked_sub(min_fresh).unwrap_or(Duration::ZERO);
```

Comment:
This should `return` as well.

Reply:
Thanks, probably that why `request-cache-force-cache.any.worker.html` failed